src/runtime/RegExpObject.cpp

```diff
@@ -175,6 +176,9 @@ void RegExpObject::init(ExecutionState& state, String* source, String* option)
 {
     parseOption(state, option);
     this->internalInit(state, source);
+#ifdef ESCARGOT_ENABLE_ES2015
+    this->set(state, ObjectPropertyName(state.context()->staticStrings().match), Value(true), this);
```

Comment:
This should be defined [here](https://github.com/pando-project/escargot/pull/157/files#diff-39a9c0d27df94495ce8b17d1bcb0b275R260) as well.